tests/pgsql-bug-6080-probe-test-01/test.yaml

```diff
@@ -0,0 +1,44 @@
+args:
+- -k none
+
+checks:
+- filter:
+    count: 0
+    match:
+      dest_ip: 172.16.4.19
+      dest_port: 5432
+      event_type: pgsql
+      pgsql.tx_id: 1
+      proto: TCP
+      src_ip: 172.16.1.1
+      src_port: 1050
+- filter:
+    count: 0
+    match:
+      app_proto: pgsql
```

Comment:
We do _not_ want the flow to be recognized as pgsql, do we ?

Reply:
So, it should be there is 1 event_type flow, and 0 with `app_proto: pgsql` and no other fields to test